rmw_dds_common/src/security.cpp

```diff
@@ -40,9 +40,9 @@ bool get_security_files(
   };
 
   for (const std::pair<const std::string, std::string> & el : required_files) {
-    rcpputils::fs::path full_path(secure_root);
+    std::filesystem::path full_path(secure_root);
     full_path /= el.second;
-    if (!full_path.is_regular_file()) {
+    if (!full_path.has_filename()) {
       result.clear();
```

Comment:
Because std::filesystem::path does not have is_regular_file member function.
We can use another function (has_filename) for our purpose.